xchange-stream-coinbasepro/src/main/java/info/bitrich/xchangestream/coinbasepro/dto/CoinbaseProWebSocketTransaction.java

```diff
@@ -367,7 +404,19 @@ public String toString() {
     if (profileId != null) sb.append(", profileId='").append(profileId).append('\'');
     if (takerUserId != null) sb.append(", takerUserId='").append(takerUserId).append('\'');
     if (takerProfileId != null) sb.append(", takerProfileId='").append(takerProfileId).append('\'');
+    if (channels != null) sb.append(", channels='").append(channels).append('\'');
     sb.append('}');
     return sb.toString();
   }
+
+  public BigDecimal getFee() {
```

Comment:
what happens when order is fully executed (or canceled) and we get 'done' type?

Reply:
web socket transaction contains only differences in the order. Done message does not contribute to a "fee" field. Only Match message does it.